Vocable/SwiftUI/Gaze Button/GazeButtonStyleConfiguration.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  GazeButtonStyleConfiguration.swift
+//  Vocable
+//
+//  Created by Robert Moyer on 4/7/22.
+//  Copyright © 2022 WillowTree. All rights reserved.
+//
+
+import SwiftUI
+
+/// The properties of a ``GazeButton`` instance.
+///
+/// When you define a custom button style by creating a type that conforms to
+/// the ``GazeButtonStyle`` protocol, you implement the
+/// ``GazeButtonStyle/makeBody(_:)`` method. That method takes a
+/// `GazeButtonStyleConfiguration` input that has the information you need
+/// to define the behavior and appearance of a ``GazeButton``.
+///
+/// The configuration structure's ``label-swift.property`` reflects the
+/// button's content, which might be the value that you supply to the
+/// `label` parameter of the ``GazeButton/init(minimumGazeDuration:role:action:label:)`` initializer.
+/// Alternatively, it could be another view that SwiftUI builds from an
+/// initializer that takes a string input, like ``GazeButton/init(_:minimumGazeDuration:role:action:)``.
```

Comment:
Looks like this function signature changed and this is now generating a warning

```suggestion
/// initializer that takes a string input, like ``GazeButton/init(minimumGazeDuration:role:action:label:)``.
```

Reply:
Nice catch! I think the reason it's throwing a warning is because this is an ambiguous reference since the initializers that accept a `LocalizedStringKey` and a `StringProtocol` each have the same signature. DocC documentation tells me that in this case it should generate a random identifier to disambiguate, but for some reason it's not doing that here 🤔 - I will look into this 🕵️‍♂️

Reply:
Fixed ✅